change from CircleCI to TravisCI

rjw57 opened the pull request:
We have a simple set up and have no need for the private repo support.

rjw57 committed:
remove circleci configuration

rjw57 committed:
add .nojekyll file into built docs

rjw57 committed:
add tox testenv to build docs

rjw57 committed:
add basic travis configuration

abrahammartin commented on .travis.yml:
```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: python
+python:
+    - "3.4"
```
shouldn't we test against 3.6? We use 3.6 in our dockerfile, no?

rjw57 replied:
Yes, this was a hang over from when we were using the standard GCE image. Will change.

abrahammartin reviewed (commented):
looks good except for a minor comment

rjw57 commented:
Huh - Apparently I changed my mind on Travis FSR since I closed this PR and deleted the branch. I don't recall why now :(. Perhaps it was an attempt to get the coveralls bot to work?
